pkg/util/age.go

```diff
@@ -0,0 +1,30 @@
+// Copyright © 2018 The Knative Authors
```

Comment:
I'm not in love with "util" as a name of things. it seems like the thing people name a bin where they stick things they don't know where else to put.